libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -500,7 +575,7 @@ const IntlAttributeFilterButton = withContexts(injectIntl(AttributeFilterButtonC
  */
 export const AttributeFilterButton: React.FC<IAttributeFilterButtonOwnProps> = (props) => {
     return (
-        <IntlWrapper>
+        <IntlWrapper locale={props.locale || "en-US"}>
```

Comment:
Either use `DefaultLocale` instead of `"en-US"` or better yet: do not provide the `||` part, the `IntlWrapper` will internally use the `DefaultLocale` if not provided with a locale.